todo_app/data/view_model.py

```diff
@@ -0,0 +1,31 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def done_items(self):
+        result = []
+        for i in self.items:
+            if i.status == "Done":
+                result.append(i)
+        return result
```

Comment:
Not a suggestion but this can be a nice opportunity to compare to the list comprehension approach here:
```python
        result = [i for i in self.items if i.status == "Done"]
        return result
```